app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'user_id'
+  belongs_to :post
+
+  def increment_post_comments_counter
+    post.increment!(:comment_counter)
+  end
```

Comment:
[OPTIONAL] Same here, you use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) -* after_save to call the `#increment_post_comments_counter`